python/pyspark/sql/dataframe.py

```diff
@@ -2192,6 +2193,51 @@ def toPandas(self):
                         _check_series_convert_timestamps_local_tz(pdf[field.name], timezone)
             return pdf
 
+    def mapPartitionsInPandas(self, udf):
+        """
+        Maps each partition of the current :class:`DataFrame` using a pandas udf and returns
+        the result as a `DataFrame`.
+
+        The user-defined function should take an iterator of `pandas.DataFrame`s and return another
```

Comment:
Yea, I will make a separate PR to document and fix the doc.. I was actually waiting for https://github.com/apache/spark/commit/8299600575024ca81127b7bf8ef48ae11fdd0594 since it shares `SCALAR_ITER`. But .. let just me do it separately